composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">=5.4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4|^5",
+        "phpunit/phpunit": "^4.8|^5.7",
```

Comment:
Any reason not to use the latest versions here? It's only a `require-dev` for this package, so compatibility with other constraints isn't required.